src/ConstantProduct.sol

```diff
@@ -169,18 +227,18 @@ contract ConstantProduct is IConditionalOrderGenerator {
      * the logic inside the original function, it frees up some stack slots and
      * reduces "stack too deep" issues.
      * @param owner the contract who is the owner of the order
-     * @param staticInput the static input for all discrete orders cut from this
-     * conditional order
+     * @param staticInput the trading parameters of all discrete orders cut
+     * from this conditional order
      * @return order the tradeable order for submission to the CoW Protocol API
      */
     function _getTradeableOrder(address owner, bytes calldata staticInput)
```

Comment:
nit: not sure if you wanted to keep `staticInput` at this point, or if it changes in a subsequent PR.

Reply:
Removed. I still had to keep a few occurrences because `@inheritdoc IConditionalOrder` enforces the parameter name, however these are going to be removed in a later PR.